### PR TITLE
Add docs for `memory_reset_peak_usage()`

### PR DESCRIPTION
**Pull request by realFlowControl**
Hi there,

this will add documentation for `memory_reset_peak_usage()` as found in #1803

Kind regards
Florian

### CODE REVIEW

File: reference/info/functions/memory-reset-peak-usage.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  <methodsynopsis>
+   <type>void</type><methodname>memory_reset_peak_usage</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Resets the peak memory usage returned by the
+   <function>memory_get_peak_usage</function> function.
+  </para>
+ </refsect1>
+
```
Comment (saundefined) [inline reference/info/functions/memory-reset-peak-usage.xml:20]: Thanks! Add please a parameters section

```
 <refsect1 role="parameters">
  &reftitle.parameters;
  &no.function.parameters;
 </refsect1>
```

**Review by cmb69 — APPROVED** [review]
Thank you!

File: reference/info/functions/memory-reset-peak-usage.xml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ <refsect1 role="examples">
+  &reftitle.examples;
+  <para>
+   <example>
+   <title><function>memory_reset_peak_usage</function> example</title>
+   <programlisting role="php">
+<![CDATA[
+<?php
+// This is only an example, the numbers below will
+// differ depending on your system
```
Comment (saundefined) [inline reference/info/functions/memory-reset-peak-usage.xml:42]: Not sure, but maybe remove it, add `&example.outputs.similar;` section,
and move into it function's output?

@cmb69 that do you think?

<details>
  <summary>Example</summary>
  
```
<programlisting role="php">
<![CDATA[
<?php

var_dump(memory_get_peak_usage());

$a = str_repeat("Hello", 424242);

var_dump(memory_get_peak_usage());

unset($a);
memory_reset_peak_usage();

$a = str_repeat("Hello", 2424);
var_dump(memory_get_peak_usage());
]]>
</programlisting>
&example.outputs.similar;
<screen>
<![CDATA[
int(422440)
int(2508672)
int(399208)
]]>
</screen>
```
  
</details>

Comment (realFlowControl) [reply reference/info/functions/memory-reset-peak-usage.xml]: Did the suggested changes, looks nicer IMHO :+1:

**Review by saundefined — APPROVED** [review]

File: reference/info/functions/memory-reset-peak-usage.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  <para>
+   <simplelist>
+    <member><function>memory_get_peak_usage</function></member>
+   </simplelist>
+  </para>
+ </refsect1>
+
+</refentry>
+
+<!-- Keep this comment at the end of the file
```
Comment (Girgias) [inline reference/info/functions/memory-reset-peak-usage.xml:79]:
```suggestion
</refentry>
<!-- Keep this comment at the end of the file
```

**Pull request merged**